src/OrchardCore.Modules/Orchard.Themes/Orchard.Themes.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\OrchardCore\Microsoft.AspNetCore.Modules.Abstractions\Microsoft.AspNetCore.Modules.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.AsModule\OrchardCore.AsModule.csproj" />
```

Comment:
This is what i did for our 4 themes in `OrchardCore.Themes`. But here `Orchard.Themes` is a **Module** which has a `Module.txt` manifest file.